apps/retrospective/models/sprint_member_task.go

```diff
@@ -39,7 +39,7 @@ const (
 type SprintMemberTask struct {
 	gorm.Model
 	SprintMember     SprintMember
-	SprintMemberID   uint                 `gorm:"not null"`
+	SprintMemberID   uint `gorm:"not null"`
```

Comment:
formatting issue

Reply:
trailing space in
```
// MemberTaskRole
const (
	Implementor MemberTaskRole = iota
	Reviewer    
	Validator   
)
```

Reply:
Formatting is as suggested by GoLint

Also, I cannot find any trailing space in this file.